test/e2e/bindata/assets/03_clusterrole.yaml

```diff
@@ -1,7 +1,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: descheduler-operator
+  name: openshift-descheduler-operator
```

Comment:
The naming needs to stay identical to what's used in https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/manifests/stable/cluster-kube-descheduler-operator.v4.14.0.clusterserviceversion.yaml#L111-L195. So we test what we deliver. At best, the e2e test should read the CSV file and extract the naming and the list of RBAC rules from it. So there's no disconnection. Otherwise, everytime the list of rbac rules in the CSV file gets updated, we need to manually mirror the changes here.

Reply:
Would be cool, but won't be doing that one in this PR :D